src/main/java/org/cbioportal/model/SurvivalEvent.java

```diff
@@ -0,0 +1,61 @@
+package org.cbioportal.model;
+
+import java.util.List;
+import java.util.function.ToIntFunction;
+
+public class SurvivalEvent {
+    List<ClinicalEvent> startClinicalEventsMeta;
```

Comment:
are these methods suppose to be package private?

Reply:
to be removed

Reply:
removed